runtime/module.go

```diff
@@ -113,12 +114,6 @@ func ProvideApp() (
 		return nil, nil, nil, nil, nil, nil, nil, nil, nil, err
 	}
 
-	// validate the signing context to make sure that messages are properly configured
-	// with cosmos.msg.v1.signer
-	if err := interfaceRegistry.SigningContext().Validate(); err != nil {
```

Comment:
Why no validation?

Reply:
With current app wiring and depinject, I couldn't find anyway to call `DefineCustomGetSigners` *after* an invoker is called.

Reply:
Interesting... can we call validate in an invoker?

Reply:
Yes but `DefineCustomGetSigners` is also called in an invoker and order matters, right?